shared_model/builders/protobuf/builder_templates/proposal_template.hpp

```diff
@@ -78,10 +74,12 @@ namespace shared_model {
             [&](auto &proposal) { proposal.set_height(height); });
       }
 
-      auto transactions(const TransactionContainer &transactions) const {
+      template <class T>
+      auto transactions(const T &transactions) const {
         return transform<Transactions>([&](auto &proposal) {
           for (const auto &tx : transactions) {
-            proposal.mutable_transactions()->Add(tx->getTransport());
+            new (proposal.add_transactions())
```

Comment:
Here the point is not about placement new. `mutable_transactions()->Add(tx->getTransport())` did not work here. So I rewrote it. It can be easily rewritten with 
```C++
auto new_tx = proposal.add_transactions();
new_tx->CopyFrom(tx.getTransport());
tx.
```